cmd/nerdctl/main.go

```diff
@@ -295,6 +295,7 @@ Config file ($NERDCTL_TOML): %s
 		image.NewTagCommand(),
 		image.NewRmiCommand(),
 		image.NewHistoryCommand(),
+		image.NewSquashCommand(),
```

Comment:
Not a fan of inflating the number of the top-level commands

Reply:
deleted